source/data-modeling/documents.txt

```diff
@@ -1,8 +1,8 @@
 .. _mongoid-modeling-documents:
 
-=========
-Documents
-=========
+===========================
+Ruby Document Object Module
```

Comment:
S: Remove ruby
A better option would be just `Document Module` or `Document Model Class Module` if it needs to be longer

Reply:
How about "Include the Document Module in Your Object Class"? Want to try something more meaningful than a longer name

Reply:
`Include the Document Module in Your Model`?